ansible/group_vars/all.yml

```diff
@@ -4,8 +4,12 @@
 #
 # localBuild - requires make release to have been run to build local binaries
 # packageManager - will install packages from your distribution using yum/dnf/apt
+# ghReleases - will download binaries from github releases
 source_type: localBuild
 
+# This version will be used only when installing kubernetes from github releases
+kubernetes_version: v1.0.4
```

Comment:
I was able to grab latest version from github using standard shell tools:

```bash
curl -sI https://github.com/kubernetes/kubernetes/releases/latest | grep Location | rev | cut -d "/" -f 1 | rev
```
I'm going to try your solution with this and if successful add another commit.